### DISCUSSION TITLE
작동이 잘되는지 모르겠네요...

### DISCUSSION THREAD

**Discussion by sH1222J** [changdoc/naver-vaccine-macro#159] [Q&A] [answered]
이런 좋은 프로그램을 이제 발견하였는데 그 사이 창과 방패의 싸움이 있던것같아 좀 바뀐게 있으신것같네요...

사용 방법
크롬 확장 프로그램에서 [NAVER Vaccine MACRO] 설치를 진행합니다.
네이버 웨일 브라우저나 최신 버전을 바로 적용하기 위해서는 개발자 모드로 코드를 받으시고 직접 설치하시면 됩니다.
(최신 버전 다운로드 방법) https://github.com/changdoc/naver-vaccine-macro/issues/29
(크롬 개발자 모드 관련 설명 블로그) https://trend21c.tistory.com/1030
https://v-search.nid.naver.com/home 네이버 우리 동네 백신 알림 페이지에 접속하여 네이버 인증서를 미리 발급받도록 합니다.
백신 접종을 희망하는 병원을 검색하여 알림 신청을 해둡니다.
잔여 백신이 발생되어 알림을 받게 되면 네이버 톡톡에 [네이버 우리 동네 백신 알림]을 통해 접종기관 알림 전달됩니다.
[네이버 우리 동네 백신 알림] 대화방에서 원하는 접종 기관에서 온 메시지에 있는 [지금 신청하기] 링크를 눌러 이동합니다. (https://v-search.nid.naver.com/reservation/standby?orgCd=XXXX&sid=XXXXX 형태)
알림을 받기 전에 원하는 병원 예약 신청 페이지 url 알아내는 방법은 별도로 작성해두었습니다.
https://github.com/changdoc/naver-vaccine-macro/issues/4
병원 예약 신청 페이지 url 목록 생성 방법도 작성해두었습니다.
https://github.com/changdoc/naver-vaccine-macro/issues/11
https://v-search.nid.naver.com/reservation/info?key=XXXX 형태의 url, 예약 신청 페이지가 정상적으로 노출되는지 확인합니다.
'자동 예약 시작' 버튼을 누르면 동작이 시작됩니다.
여러 페이지를 띄워놓고 수행한다면 한 곳에서만 '자동 예약 시작' 버튼을 누르고 나머지 페이지에서는 브라우저에서 새로 고침 하면 동작됩니다.
페이지를 계속 새로 고침 하며 잔여 수량이 있을 경우 예약을 시도합니다. (test mode 체크된 상태에서는 실제로 시도하지 않습니다.)
예약이 시도 시 소리와 함께 결과 화면으로 이동합니다.

에서 "'자동 예약 시작' 버튼을 누르면 동작이 시작됩니다."라고 명시되어있는데 자동 예약 시작버튼이 안보이네요...
방법이 바뀐건지 
graphql result data를 추출해서 해보았는데도 이렇게 하면 작동이 되는게 맞는건지 궁금합니다...
백신이 나오면 소리가 나온다는데 작동하는건질 모르겟네요
![화면 캡처 2021-07-30 175519](https://user-images.githubusercontent.com/28824568/127628489-e0e29222-324a-4cf6-9d38-9e01cae2ec10.png)

**Comment by changdoc** [comment, marked as answer]
매크로 프로그램에 생기는게 아니라

https://v-search.nid.naver.com/reservation/info?key=XXXX 형태의 url, 예약 신청 페이지가 정상적으로 노출되는지 확인합니다.
'자동 예약 시작' 버튼을 누르면 동작이 시작됩니다.

이 부분에서 [https://v-search.nid.naver.com/reservation/info?key=XXXX 형태의 url, 예약 신청 페이지] 예약 신청 페이지에 버튼이 생깁니다.

![image](https://user-images.githubusercontent.com/1879494/127634562-51c33719-be36-424b-8527-8152cbf4f4d3.png)

이렇게요

**Reply by sH1222J** [reply]
아 자세히 보니 버튼이 있군요.
감사합니다!
